tests/README.md

```diff
@@ -4,28 +4,14 @@
 
 **Set up**
 
-1. Install phpunit either globally or in the project with composer by running `composer install`.
-2. Install and run mysql server either locally on in docker.
-3. Run script `tests/bin/install.sh`:
-
-	Pass db accesss arguments as params e.g. `tests/bin/install.sh db_name user password db_host`.
-	Example command if running DB in docker and port 5678 is exposed by docker container:
-	```
-	tests/bin/install.sh test_gateway root wordpress 127.0.0.1:5678
-	```
+1. Start the local environment (_docker containers_) with:
+```
+npm run up
+```
 
 **Running the tests**
 
-If phpunit is installed with composer run the tests with command:
+Run the tests with:
 ```
 npm run test:php
```

Comment:
I guess it's still interesting to give the option to run tests without Docker. I tested here and it worked.

You could copy the same instructions from [WCPay](https://github.com/Automattic/woocommerce-payments/tree/trunk/tests), just adapting it a little, like removing the `docker-compose up -d`. Do you think it's worth it?

Reply:
I think it would be confusing.
It's possible to have the local environment working correctly with docker, but unable to run tests locally... for example in my previous laptop I get this when trying to run the tests without docker:
<img width="1154" alt="Screen Shot 2021-02-19 at 18 54 30" src="https://user-images.githubusercontent.com/407542/108565945-59d25a80-72e4-11eb-8229-6c481ba7c42e.png">

Reply:
That's weird. I had tested before and just to confirm I tested again with the latest version of your branch and it works for me.

<img width="918" alt="Screen Shot 2021-02-19 at 19 21 24" src="https://user-images.githubusercontent.com/5509901/108567893-d6b30380-72e7-11eb-8d84-c24001ec9cae.png">

Any ideas why that is?

Reply:
Not really, I didn't spend time on it, since the idea of this PR is to make running tests easier, without the need to clone the `woocommerce` repository in the same parent folder, nor the need to have the `wordpress-tests-lib` in the tmp folder (or in some cases /private/tmp_***)